### ISSUE TITLE
Missing textures for radial LF/LFO tanks

### ISSUE THREAD

**Issue by blowfishpro** [VenVen/Stock-Revamp#63]
The textures were removed in f05d8a42561bfc1cee643ea98a07270b4c06a63e, but the models still reference them

```
Texture 'VenStockRevamp/Squad/Parts/Propulsion/RCSTANKS_CLR 1' not found!
Texture 'VenStockRevamp/Squad/Parts/Propulsion/RCSTANKS-Bump' not found!
```

**Comment by VenVen** [comment]
I cant seem to find those errors in my output log, I may have forgot to include the updated RCS tanks at the time of  that update, but i included those a day or two later.

**Comment by blowfishpro** [comment]
Yeah, I see the commit (looks like within an hour or so of when I created this issue).  I'll test when I get home.

**Comment by blowfishpro** [comment]
Yep, fixed in b3b82a04b3116e4868b5a5e8df5058c4ffb8a891
